chanutils/func.go

```diff
@@ -70,3 +72,35 @@ func All[T any](xs []T, pred func(T) bool) bool {
 
 	return true
 }
+
+// Any returns true if the passed predicate returns true for any item in the
+// slice.
+func Any[T any](xs []T, pred func(T) bool) bool {
+	for _, x := range xs {
+		x := x
```

Comment:
Yes, in this specific case it wasn't necessary, because calling a method will put the pointer on the stack, which causes the Golang loop variable "bug" not to apply. But in general, it's good practice to do this anyway (or switch to the index based access which I did).